app/models/subscription/BusinessSubscriptionDetailsModel.scala

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.subscription
+
+import models.DateModel
+import play.api.libs.json._
+
+sealed trait AccountingMethod {
+  val stringValue: String
+}
+
+case object Cash extends AccountingMethod {
+  override val stringValue = "CASH"
+}
+
+case object Accural extends AccountingMethod {
```

Comment:
This is misspelled and should be Accrual (not a comment on the value but if DES/ETMP mispelled it I would rather we only matched the value and spelled the names correctly)

Reply:
Done